hermes-console-vue/src/composables/use-stats/useStats.spec.ts

```diff
@@ -0,0 +1,127 @@
+import { expect } from 'vitest';
+import { statsResponse } from '@/dummy/stats';
+import { useStats } from '@/composables/use-stats/useStats';
+import { waitFor } from '@testing-library/vue';
+import axios from 'axios';
+import type { Mocked } from 'vitest';
+
+vitest.mock('axios');
+const mockedAxios = axios as Mocked<typeof axios>;
+
+describe('useStats', () => {
+  it('should hit expected Hermes API endpoint', async () => {
+    // given
+    mockedAxios.get.mockResolvedValueOnce({ data: statsResponse });
```

Comment:
Instead of mocking axios, use mocked server from msw library. U can look at useTopic.spec.ts how it should be done

Reply:
fixed